sql/derby/DerbyLexer.g4

```diff
@@ -353,9 +353,6 @@ XMLSERIALIZE:                                          'XMLSERIALIZE';
 YEAR:                                                  'YEAR';
 
 
-DUMMY: 'DUMMY';
```

Comment:
So, finally it was incorrect token since you removed it at all?

Reply:
> So, finally it was incorrect token since you removed it at all?

it was WIP token